utilities/rocAL/rocAL_unittests/rocAL_unittests.cpp

```diff
@@ -747,7 +750,7 @@ int test(int test_case, int reader_type, const char *path, const char *outName,
     {
         std::cout << ">>>>>>> Running "
                   << "rocalResizeCropMirrorFixed" << std::endl;
-        image1 = rocalResizeCropMirrorFixed(handle, image0, 300, 300, true, 250, 250, 0);
+        image1 = rocalResizeCropMirrorFixed(handle, image0, 300, 300, true, 250, 250, mirror);
```

Comment:
Why for fixed we are getting rocalintparam? We should be getting unit ryt?

Reply:
The 2 API rocalCropMirrorNormalize and rocalResizeCropMirrorFixed, both are accepting mirrors as RocalIntParam only